indexing-service/src/main/java/io/druid/indexing/overlord/ForkingTaskRunnerFactory.java

```diff
@@ -34,6 +34,7 @@
  */
 public class ForkingTaskRunnerFactory implements TaskRunnerFactory<ForkingTaskRunner>
 {
+  public static final String TYPE_NAME = "local";
```

Comment:
this is a bit odd imo, it's only a "local" runner on the overlord. On the middleManager it's the normal runner.